frontend/src/components/Modals/DetailsButtonModal.tsx

```diff
@@ -69,21 +74,35 @@ const DetailItem = ({
   title,
   description,
   message,
+  schema,
+  instance,
   borderClass,
 }: {
   title: string;
   description: string;
   message: string;
+  schema: Record<string, unknown> | boolean;
+  instance: unknown;
   borderClass: string;
 }) => {
+  const schemaFormatted = JSON.stringify(schema, null, 2);
+  const instanceFormatted = JSON.stringify(instance, null, 2);
   return (
     <div className="col">
       <div className={`card mb-3 ${borderClass}`}>
         <div className="card-body">
-          <h5 className="card-title">{title}</h5>
-          <p className="card-text">{description}</p>
+          <h5 className="card-title">Case : {title}</h5>
```

Comment:
These should probably use `<label>`, but I don't really want to spend much time here -- I wouldn't really have been comfortable merging this PR as-is rather than simply doing the work to use the schema/instance widget which I thought I was clear about in https://github.com/bowtie-json-schema/bowtie/issues/871#issuecomment-1954413824 -- I said:

> I think at very least we should reuse/improve the schema/instance widget

I'll try to be even clearer next time.

Please fix the minor inconsistencies (which I'll leave suggestions on in a second) and then I'll merge this one time, but really this is a bit too underpolished for comfort -- the existing UI is pretty unpolished but I prefer not churning.

Reply:
I approached this issue as a bug fix doing only as much as is needed to bringing in new data without changing the surrounding code. It was tagged as good-first-issue and I assumed it to mean no refactoring or no changing more than what is needed. Now that I spent a few days working with the community and mentors, I understand a bit more about how to work with  feedback and comments.

Reply:
Note that it was tagged [enhancement](https://github.com/bowtie-json-schema/bowtie/issues/871).

"Don't change more than what's needed" is always a good rule (for any kind of change), but in general for enhancements that means "don't invent a halfway thing unless it's clearly better, and definitely not if we immediately switch to the real one right after, in that case, just do the real work".

Reply:
Thanks for the feedback.